### ISSUE TITLE
Wikipedia Image URL collect test fails

### ISSUE THREAD

**Issue by harman28** [kenpratt/wikipedia-client#53]
Latest builds show a regular failure:
```
Wikipedia::Client.find page (Edsger_Dijkstra)
  should collect the image urls (FAILED - 1)
```

The URL list probably just needs to be updated.

Ideally, HTTP requests made in tests should be stubbed, so the code doesn't need to keep getting updated when someone makes a change on Wikipedia. But that's a bigger task, and probably merits a different issue.

**Comment by pietromenna** [comment]
Hi @harman28 , this is what I tried to point in #47 . There are many improvements to be done. All help is welcome!

**Comment by harman28** [comment]
@pietromenna There's one thing I'm confused about. How is Travis marking these builds as [passed](https://travis-ci.org/kenpratt/wikipedia-client/builds/235054310), if there's a test that always fails? Has that test been exempted somehow?

**Comment by harman28** [comment]
The ^issue I spoke out is outlined in #56 and a fix is proposed in #57.

Closing this issue as it was fixed in #55
